BmcApi/lib/pnap_bmc_api/models/quota_edit_limit_request_details.rb

```diff
@@ -111,10 +111,6 @@ def list_invalid_properties
         invalid_properties.push('invalid value for "reason", reason cannot be nil.')
       end
 
-      # This pattern is being manually modified.
-      # (?m) was originally (?s) -- it functions the same, except Ruby doesn't understand (?s)
-      # Information here:
-      #   - https://www.rexegg.com/regex-ruby.php
       pattern = Regexp.new(/^(?m)(?!\s*$).+/)
```

Comment:
Checking, but you didn't need to change it to `(?m)` manually right?

Reply:
yes i had to, develop was overriding the change

Reply:
added back the comment to make it obvious that this (and other changes like it) are manual changes